apps/dashboard/src/model-assessment/__mock_data__/adultCensus.ts

```diff
@@ -25,6 +25,1475 @@ export const adultCensusWithFairnessDataset: IDataset = {
     "native-country"
   ],
   class_names: ["<=50K", ">50K"],
+  data_balance_measures: {
```

Comment:
hmm I see errors like "Identifier 'data_balance_measures' is not in camel case"
I wonder if you need to make these camel case then, like dataBalanceMeasures.  It seems like some variables like class_names are not camel case though.  I'm not sure why it doesn't complain about them.  Perhaps it's whitelisted somewhere.